ui/src/app/(index)/_components/BeforeLogin.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from '@/components/ui/button';
+import Link from 'next/link';
+import { ArrowRight } from 'lucide-react';
+import UHGroupingsInfo from '@/components/UHGroupingsInfo';
+
+const BeforeLogin = () => {
+    return (
+        <main className="bg-seafoam pb-10">
+            <UHGroupingsInfo variant="home"/>
+            <div className="row">
+                <div className="text-center">
+                    <LearnMoreButton/>
+                </div>
+            </div>
+        </main>
+    );
+};
+
+const LearnMoreButton = () => {
+    return (
+        <Link href="https://uhawaii.atlassian.net/wiki/spaces/UHIAM/pages/13403213/UH+Groupings">
+            <Button size="lg" variant="default">
+                Learn More <ArrowRight className="ml-1"/>
+            </Button>
+        </Link>
+    );
+};
```

Comment:
I would just put the LearnMoreButton as the code itself rather than split it into it's own component. The button is very simply anyway. Also use `<a>` instead of `<Link>` because `<Link>` is designed for visiting a route rather than a link to another website. Thanks!